shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -25,6 +25,12 @@
     public static final String HDFS_READ_BYTES = "tier2_read_bytes";          // Counter
     public static final String HDFS_WRITE_BYTES = "tier2_write_bytes";        // Counter
 
+    //filesystem stats
+    public static final String FILESYSTEM_READ_LATENCY = "tier2_read_latency_ms";   // Timer
```

Comment:
How is this different from the above (with HDFS)? Please either:
1. Rename the variables to something more generic and reuse them.
2. Update their values to clearly indicate "hdfs", "filesystem", etc.